code/optimize/src/test/java/com/adobe/marketing/mobile/optimize/OptimizeTests.java

```diff
@@ -90,475 +69,548 @@ public void test_extensionVersion() {
-                .setEventData(eventData).build();
-        callbackWithError.call(event);
-
-        assertNull(responseError);
-        assertNull(responseMap);
+        try (MockedStatic<MobileCore> mobileCoreMockedStatic = Mockito.mockStatic(MobileCore.class)) {
+            // test
+            Optimize.onPropositionsUpdate(new AdobeCallbackWithError<Map<DecisionScope, Proposition>>() {
+                @Override
+                public void fail(final AdobeError adobeError) {
+                    responseError = adobeError;
+                }
+
+                @Override
+                public void call(final Map<DecisionScope, Proposition> propositionsMap) {
+                    responseMap = propositionsMap;
+                }
+            });
+
+            //verify
+            final ArgumentCaptor<AdobeCallbackWithError<Event>> callbackCaptor = ArgumentCaptor.forClass(AdobeCallbackWithError.class);
+            mobileCoreMockedStatic.verify(() -> MobileCore.registerEventListener(eq("com.adobe.eventType.optimize"), eq("com.adobe.eventSource.notification"),
+                    callbackCaptor.capture()), times(1));
+            final AdobeCallbackWithError<Event> callbackWithError = callbackCaptor.getValue();
+
+            final List<Map<String, Object>> propositionsList = new ArrayList<>();
+            propositionsList.add(null);
+
+            final Map<String, Object> eventData = new HashMap<>();
+            eventData.put("propositions", propositionsList);
+            final Event event = new Event.Builder("Optimize Notification", "com.adobe.eventType.optimize", "com.adobe.eventSource.notification")
+                    .setEventData(eventData).build();
+            callbackWithError.call(event);
+
+            assertNull(responseError);
+            assertNull(responseMap);
+        }
     }
 
     @Test
     public void test_clearCachedPropositions() {
-        // test
-        Optimize.clearCachedPropositions();
-
-        //verify
-        PowerMockito.verifyStatic(MobileCore.class, Mockito.times(1));
-        final ArgumentCaptor<Event> eventCaptor = ArgumentCaptor.forClass(Event.class);
-        MobileCore.dispatchEvent(eventCaptor.capture(), any(ExtensionErrorCallback.class));
-        final Event event = eventCaptor.getValue();
-
-        assertNotNull(event);
-        assertEquals("com.adobe.eventType.optimize".toLowerCase(), event.getType());
-        assertEquals("com.adobe.eventSource.requestReset".toLowerCase(), event.getSource());
-        assertTrue(event.getEventData().isEmpty());
+        try (MockedStatic<MobileCore> mobileCoreMockedStatic = Mockito.mockStatic(MobileCore.class)) {
+            // test
+            Optimize.clearCachedPropositions();
+
+            //verify
+            final ArgumentCaptor<Event> eventCaptor = ArgumentCaptor.forClass(Event.class);
+            mobileCoreMockedStatic.verify(() -> MobileCore.dispatchEvent(eventCaptor.capture()), times(1));
```

Comment:
nit: `times(1)` is redundant here as `mobileCoreMockedStatic.verify(() -> MobileCore.dispatchEvent(eventCaptor.capture()));` verifies that certain behavior happened once. It is 
alias to verify(mock, times(1)). Check this in other tests as well.